config/docker/dt-validation/Dockerfile

```diff
@@ -22,3 +22,5 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     libyaml-dev \
     pkg-config \
     python3-yaml
+
+RUN pip3 install --system yamllint
```

Comment:
How about installing the `yamllint` Debian package instead?
```
gtucker@tincan:~$ apt show yamllint
Package: yamllint
Version: 1.26.0-2
Priority: optional
Section: devel
Maintainer: Debian Python Team <team+python@tracker.debian.org>
Installed-Size: 189 kB
Depends: python3-pathspec, python3-pkg-resources, python3-yaml, python3:any
Homepage: https://github.com/adrienverge/yamllint
Download-Size: 39.5 kB
APT-Sources: http://deb.debian.org/debian bullseye/main amd64 Packages
Description: Linter for YAML files
 yamllint does not only check for syntax validity, but for weirdnesses like key
 repetition and cosmetic problems such as lines length, trailing spaces,
 indentation, etc.
```

Reply:
Huh, that wasn't coming up when I was searching yesterday.